src/init-ceph.in

```diff
@@ -50,6 +50,18 @@ if [ -n "$CEPH_BIN" ] && [ -n "$CEPH_ROOT" ] && [ -n "$CEPH_BUILD_DIR" ]; then
   ASSUME_DEV=1
 fi
 
+if [ `uname` = FreeBSD ]; then
+  GETOPT=/usr/local/bin/getopt
+  ULIMIT="limit descriptors"
+else
+  GETOPT=getoption
```

Comment:
@tchaikov 
Eh yes it should. I did fix this, 
And removed the ULIMIT stuff, since I wrongly thought /bin/sh was different in this.

In the mean time I've found some more enhancements.
I'll put this on DNM and fix.

Note that I have a three system cluster FreeBSD system up and running.... 8-D
But it also means I have a hard time tracking all the fixes to the packages and rc.d files. :(